src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyCost = 40;
```

Comment:
Consider using a constant for `dailyCost` with a descriptive name to explain what this value represents, such as `DAILY_RENTAL_RATE`. This improves code readability and maintainability.